TrackParamFilter/sections/specific.txt

```diff
@@ -272,9 +272,14 @@ $removeparam=wgexpiry,domain=ibood.com|decathlon.pl
 ||yahoo.co.jp^$removeparam=sc_e
 ! https://github.com/AdguardTeam/AdguardFilters/issues/161441
 ||happymail.co.jp^$removeparam=Log
-! Youtube
-$removeparam=embeds_referring_euri,domain=youtubekids.com|youtube-nocookie.com|youtube.com
-||youtube.com^$removeparam=pp
+! YouTube
+$removeparam=bp,domain=youtube.com|youtu.be|youtube-nocookie.com|youtubekids.com
```

Comment:
Still cannot reproduce link with this parameter

```suggestion
```

Reply:
> Still cannot reproduce link with this parameter

On the "Explore" section of the youtube homepage.

<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/70459964/f775970e-8319-45d7-bd74-1ad7a797a394)


</details>